### ISSUE TITLE
Create a variable style ordered list

### ISSUE THREAD

**Issue by nbaxley-uiuc** [ATLAS-Illinois/Illinois-Framework-Theme#467]
- [ ] Add the CSS below to the theme
- [ ] Add the variable-ol style to the Styles configuration labeled as Variable numbering
`ol.variable-ol|Variable numbering`

When nesting ordered list, the user should have the option to alter the numbering style between decimal and alpha.  This is not straightforward in CSS, but using a class like the one below will achieve it.  We can use the Styles configuration dropdown to apply these and it seems to be applied to the top level OL only properly.

```
ol.variable-ol {
	list-style: decimal;
}
ol.variable-ol ol {
	list-style: lower-alpha; 
}
ol.variable-ol ol ol {
	list-style: lower-roman;
}
ol.variable-ol ol ol ol {
	list-style: decimal; 
}
```

**Comment by nbaxley-uiuc** [comment]
Expected formatting is shown below
![image](https://user-images.githubusercontent.com/62560479/137170875-4e8be31f-7dee-42aa-8364-86b44dd84604.png)

**Comment by nbaxley-uiuc** [comment]
@magjarvis I think I have the CSS correct for this, but it needs testing and integrating into the theme as well as getting the class integrated into the CK Editor for Basic HTML

**Comment by nbaxley-uiuc** [comment]
This should be solved by #535

**Comment by nbaxley-uiuc** [comment]
Fixed with #535

**Comment by mwllr** [comment]
List formatting was added to components 2.5 released in December 2021.
https://cdn.toolkit.illinois.edu/2.5/examples/stylesheets/typography/lists/index.html
@nbaxley-uiuc
